Sources/Swiftly/Update.swift

```diff
@@ -56,7 +56,7 @@ struct Update: SwiftlyCommand {
         }
 
         print("updating \(oldToolchain) -> \(newToolchain)")
-        try await Install.execute(version: newToolchain)
+        try await Install.execute(version: newToolchain, use: false)
```

Comment:
I know update hasn't been merged yet but shouldn't `swiftly update` also default to using the newly updated toolchain.

Reply:
This is just in here to have the existing update code compile, but it'll be changed in #67. The planned logic for this is if the toolchain being updated is the one that is in use, then the newly installed one will be used, otherwise no.